daprdocs/content/en/developing-applications/building-blocks/jobs/jobs-features-concepts.md

```diff
@@ -0,0 +1,122 @@
+---
+type: docs
+title: "Features and concepts"
+linkTitle: "Features and concepts"
+weight: 2000
+description: "Learn more about the Dapr Jobs features and concepts"
+---
+
+Now that you've learned about the [jobs building block]({{< ref jobs-overview.md >}}) at a high level, let's deep dive 
+into the features and concepts included with the Dapr Jobs engine and SDKs. Dapr Jobs:
```

Comment:
```suggestion
into the features and concepts included with the Dapr Scheduler service and SDKs. Dapr Jobs:
```

Reply:
The Scheduler service is an internal implementation detail. While it's an integral part of the Jobs capability, is that something we want to get into in a document that's sharing information about the publicly facing aspects of the Jobs functionality (which may be less subject to change than the internal bits)?

Reply:
@WhitWaldo "jobs engine" is not a term used in the user-facing docs, while "Scheduler service" is clearly defined in the docs. If "Scheduler service" does not apply here, you'll need to think of another term without introducing a new one.

Reply:
I was following the language style used in the [Workflow architecture](https://docs.dapr.io/developing-applications/building-blocks/workflow/workflow-architecture/) that uses "engine" all over the place.

If "engine" isn't preferred here, how about just "into the features and concepts included in Dapr Jobs and the various SDKs."

Reply:
Workflow engine has been defined in the docs, while this would be the first time we refer to anything called a jobs engine. Let's just call it Dapr jobs for now!